packages/office-addin-lint/src/lint.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import {logErrorMessage} from "office-addin-cli";
+import * as defaults from "./defaults";
+
+function execCommand(command: string) {
+    const execSync = require('child_process').execSync;
+    const child = execSync(command, { stdio: 'inherit' });
+}
+
+export function lintCheckCommand(files: string): string {
+  const eslintCommand: string = "eslint " + files;
+  return eslintCommand;
+}
+
+export function lintCheck(files: string) {
+  console.log("Lint check for files: " + files);
+  const command = lintCheckCommand(files);
+  execCommand(command);
+}
+
+export function lintFixCommand(files: string): string {
+  const eslintCommand: string = "eslint --fix " + files;
+  return eslintCommand;
+}
+
+export function lintFix(files: string) {
+    console.log("Fixing lint issues for files: " + files);
+    const command = lintFixCommand(files);
+    execCommand(command);
+}
+
+export function prettierCommand(files: string): string {
+  const prettierFixCommand: string = "prettier --parser typescript --write " + files;
+  return prettierFixCommand;
+}
+
+export function prettier(files: string) {
+    console.log("Fixing prettier issues for files: " + files);
```

Comment:
We don't want console.log directly in the implementation. We could have it as part of the CLI, but if this is imported from another package, they could decide whether to do any console output. 

Along those lines we might want the command to capture console output and it could return it back to the CLI and the CLI could print it.

Reply:
removing console.log calls